debezium-core/src/main/java/io/debezium/relational/ColumnEditor.java

```diff
@@ -39,13 +39,12 @@
     int jdbcType();
 
     /**
-     * Get the underlying type for this column where jdbcType() itself is the generic Types.ARRAY (and in future
-     * Types.STRUCT).  If there is no underlying type information then the value is unset.
+     * Get the native type for this column.
      *
      *
-     * @return a type constant for the specific database, or -1 if not set.
+     * @return a type constant for the specific database
      */
-    int componentType();
+    int nativeType();
```

Comment:
What is a "native type"? This warrants some more description in the method docs.